executors/sshexec/volume.go

```diff
@@ -51,23 +49,18 @@ func (s *SshExecutor) VolumeCreate(host string,
 		maxPerSet = 1
 	}
 
-	// Setup volume create command
 	// There could many, many bricks which could make the command line
 	// too long.  Instead, create the volume first, then add each brick set.
 	for _, brick := range volume.Bricks[:inSet] {
 		cmd += fmt.Sprintf("%v:%v ", brick.Host, brick.Path)
 	}
 
-	// Initialize the commands with the create command
 	commands := []string{cmd}
 
-	// Now add all the commands to add the bricks
```

Comment:
Thanks for the comment @raghavendra-talur .
I still that this individual comment is bogus.
Instead, I have expanded the comment block explaining our strategy,
and separated it from the for-loop by an empty line, indicating that it
does not relate to the next line or block but to the following commands,
each of which are imho pretty idiomatic and natural to read.
(I just want to avoid the pattern
```
  // now add commands to add broicks
  s.createAddBrickCommands(...)
```
which is kind-of the definition of redundancy ;-)